[2024-02-19T18:29:59.265+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-18T00:00:00+00:00 [queued]>
[2024-02-19T18:29:59.278+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-18T00:00:00+00:00 [queued]>
[2024-02-19T18:29:59.279+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-19T18:29:59.427+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-02-18 00:00:00+00:00
[2024-02-19T18:29:59.498+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v08', 'create_postgres_table', 'scheduled__2024-02-18T00:00:00+00:00', '--job-id', '452', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpv8qy9wft']
[2024-02-19T18:29:59.500+0000] {standard_task_runner.py:88} INFO - Job 452: Subtask create_postgres_table
[2024-02-19T18:29:59.466+0000] {standard_task_runner.py:60} INFO - Started process 1167 to run task
[2024-02-19T18:29:59.819+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-18T00:00:00+00:00 [running]> on host 55ac849b3b96
[2024-02-19T18:30:00.100+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Shiv' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v08' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-18T00:00:00+00:00'
[2024-02-19T18:30:00.101+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        
[2024-02-19T18:30:00.164+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:30:00.181+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:30:00.197+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        , parameters: None
[2024-02-19T18:30:00.218+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v08, task_id=create_postgres_table, execution_date=20240218T000000, start_date=20240219T182959, end_date=20240219T183000
[2024-02-19T18:30:00.288+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-19T18:30:00.345+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
